app/controllers/preprints/provider/moderation.js

```diff
@@ -6,7 +7,6 @@ import { task } from 'ember-concurrency';
 
 import Analytics from 'ember-osf/mixins/analytics';
 
-
```

Comment:
Not a big deal but the two spaces after the imports is intentional to make it easier to scan.